frontend/src/components/pages/Landing.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Center,
+  Text,
+  Stack,
+  VStack,
+  Image,
+  Button,
+  Link,
+} from "@chakra-ui/react";
+import * as Routes from "../../constants/Routes";
+
+import JumpMathLogo from "../../assets/jump-math-logo.png";
+
+const PageLinks = (): React.ReactElement => {
```

Comment:
I hate this name, so if you have a better one please share :D

Reply:
UserDashboardLinks? idk lol

Reply:
sounds good :D